packages/typespec-azure-resource-manager/lib/models.tsp

```diff
@@ -40,7 +40,7 @@ model ResourceNameParameter<
 @armResourceInternal(Properties)
 @includeInapplicableMetadataInPayload(false)
 model TrackedResource<Properties extends {}> extends Foundations.TrackedResource {
-  @doc("The resource-specific properties for this resource.")
+  @doc("The resource-specific optional properties for this resource.")
```

Comment:
I don't think these doc updates are necessary.  The typespec and the swagger already say it is optional, adding the documentation makes swagger changes without adding any information.